apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -175,12 +175,12 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
      * disabled by the user.
      */
     const resolveInitialIDTokenEncryptionValues = (): void => {
-        if (!initialValues?.idToken?.encryption?.enabled) {
+        if (initialValues?.idToken?.encryption?.enabled === false) {
             initialValues.idToken.encryption = {
                 method: "",
```

Comment:
method should come after `algorithm` & `enabled`. Please check if eslint is properly working.

Reply:
Thanks this lint issue is fixed in commit 2706edd